nwnexplorer/SoundView.h

```diff
@@ -50,7 +50,7 @@
 #include "resource.h"
 #include "DataRef.h"
 #include "CommonView.h"
-#include <fmod.h>
+#include "include\fmod\fmod.h"

```

Comment:
Can you remove the fmod include changes? I think that'd be better for a separate PR. The code compiles as is on my machine, and I think it may be better to fix the include path in project settings instead.